packages/cdktf/lib/terraform-output.ts

```diff
@@ -5,7 +5,6 @@ import { TerraformElement } from "./terraform-element";
 import { deepMerge } from "./util";
 import { ITerraformDependable } from "./terraform-dependable";
 import { Expression } from ".";
-import { isArray } from "util";
```

Comment:
This dates to https://github.com/hashicorp/terraform-cdk/pull/1145. AFAICT the use of `util.isArray` was unintentional. `Array.isArray` is used everywhere else.